src/metatrain/experimental/nanopet/default-hypers.yaml

```diff
@@ -22,6 +22,7 @@ architecture:
     scheduler_factor: 0.8
     log_interval: 10
     checkpoint_interval: 100
+    scale_targets: true
```

Comment:
How much of a difference does this make in the end?

Reply:
On ordinary "thermal" datasets in units of eV and A, not much because the standard deviation is already close to optimal. On datasets with weird units it can make a lot of difference